sled-agent/types/Cargo.toml

```diff
@@ -28,16 +29,26 @@ serde.workspace = true
 serde_human_bytes.workspace = true
 serde_json.workspace = true
 sha3.workspace = true
+sled-hardware.workspace = true
 sled-hardware-types.workspace = true
+sled-storage.workspace = true
```

Comment:
Ah crap, this [does introduce](https://buildomat.eng.oxide.computer/wg/0/details/01JZ3SFAZK4HB8429Z517PCC81/NcW6xgFekSiiyffAFwRBC8fcq6KA0qXFc35ffJoTHuKCxRrZ/01JZ3SFXFAEYZXDWC8WJTFH51H#S10209) a libnvme.so.1 dependency to sled-agent-types, sorry...

One option is to move it to sled-storage, next to `Disk`. Another is to have a separate  "sled-agent-storage-types" crate.